Cannot run tests with mocha

When I try to run tests with mocha, it breaks with message:

> Error: Cannot find module '@captaincodeman/rdx'

This is due to incompatibilities between CommonJS / node module loading and ES modules.
Even with [esm](https://www.npmjs.com/package/esm) in the mix, it won't work.

I found this is due to rdx's `package.json` specifying: `{"type":"module", "module":"lib/index.js"}`

For compatibility I'd suggest adding `"main":"lib/index.js"` to `package.json`.

--- a/package.json
+++ b/package.json
@@ -4,6 +4,7 @@
   "description": "Small state library. Like Redux, but smaller",
   "type": "module",
   "module": "lib/index.js",
+  "main": "lib/index.js",
   "types": "typings/index.d.ts",
   "sideEffects": false,
   "files": [

